dsl/pace/dsl/dace/dace_config.py

```diff
@@ -0,0 +1,58 @@
+import dataclasses
+from typing import Callable
+
+from pace.util.global_config import getenv_bool
```

Comment:
This module still exists? You don't need to here, but we should look at deleting the unused parts of it now that it's unused.